src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -110,26 +110,42 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
   def processTezosBlocks(): Future[Done] = {
     logger.info("Processing Tezos Blocks..")
 
-    val blocksToSynchronize = tezosConf.depth match {
-      case Everything => tezosNodeOperator.getAllBlocks(tezosConf.network)
+    val blockPagesToSynchronize = tezosConf.depth match {
       case Newest => tezosNodeOperator.getBlocksNotInDatabase(tezosConf.network)
-      case Custom(n) => tezosNodeOperator.getLatestBlocks(tezosConf.network, n)
+      case Everything => tezosNodeOperator.getLatestBlocks(tezosConf.network)
+      case Custom(n) => tezosNodeOperator.getLatestBlocks(tezosConf.network, Some(n))
     }
 
-    blocksToSynchronize.flatMap {
-      blocksWithAccounts =>
-        db.run(TezosDb.writeBlocksAndCheckpointAccounts(blocksWithAccounts.toMap))
-          .andThen {
+    def storeResultsPage(results: Future[tezosNodeOperator.BlockFetchingResults]): Future[Int] =
```

Comment:
I think this function name is a bit cryptic. May I suggest something like `processBlocksPage`?

Reply:
done